src/Makefile

```diff
@@ -109,7 +109,11 @@ else
 WASAN=
 endif
 
-MYCXXFLAGS=-std=c++11 $(IDIRS) $(OPTZ) $(DEBUG) $(PSQLCH) -DGITVERSION=\"$(GIT_VERSION)\" $(WGCOV) $(WASAN)
+ifeq ($(CXX),clang++)
```

Comment:
To avoid duplicates (unfortunately the current Makefiles have a lot of this, but we can optimize moving forward) we can rewrite (keeping the same order) as:
```
MYCXXFLAGS=-std=c++11 
ifeq ($(CXX),clang++)
	MYCXXFLAGS+= -fuse-ld=lld
endif
MYCXXFLAGS+= $(IDIRS) $(OPTZ) $(DEBUG) $(PSQLCH) -DGITVERSION=\"$(GIT_VERSION)\" $(WGCOV) $(WASAN)
```
If the order of some flag is not important, it can be simplified even further

Reply:
done.